clang/lib/DPCT/APINames.inc

```diff
@@ -84,6 +84,10 @@ ENTRY(cudaGetErrorString, cudaGetErrorString, true, NO_FLAG, P0, "Successful")
 ENTRY(cudaGetLastError, cudaGetLastError, true, NO_FLAG, P0, "DPCT1010")
 ENTRY(cudaPeekAtLastError, cudaPeekAtLastError, true, NO_FLAG, P0, "DPCT1026/DPCT1027")
 
+// error handling functions of driver API
+ENTRY(cuGetErrorName, cuGetErrorName, true, NO_FLAG, P0, "Successful")
```

Comment:
I think there are  already cuGetErrorName and cuGetErrorString in https://github.com/oneapi-src/SYCLomatic/blob/f2c0cab249c63dd0d729dd0a08e47bd3021ea02e/clang/lib/DPCT/APINames.inc#L1567

Reply:
Thanks for pointing that out @wenhuiNi.